ui/component/or-dashboard-builder/src/widgets/or-image-widget.ts

```diff
@@ -0,0 +1,366 @@
+import manager, {DefaultColor5, DefaultColor4, DefaultColor2, DefaultColor3, Util } from "@openremote/core";
+import { Asset, Attribute, AttributeRef, DashboardWidget } from "@openremote/model";
+import { showSnackbar } from "@openremote/or-mwc-components/or-mwc-snackbar";
+import { i18next } from "@openremote/or-translate";
+import { css, html, LitElement, TemplateResult, unsafeCSS } from "lit";
+import { customElement, property, state, query } from "lit/decorators.js";
+import { OrWidgetConfig, OrWidgetEntity } from "./or-base-widget";
+import { style } from "../style";
+import {debounce} from "lodash";
+import { SettingsPanelType, widgetSettingsStyling } from "../or-dashboard-settingspanel";
+import { InputType, OrInputChangedEvent } from "@openremote/or-mwc-components/or-mwc-input";
+
+export interface ImageWidgetConfig extends OrWidgetConfig {
+    displayName: string;
+    attributeRefs: AttributeRef[];
+    xCoordinatesMap: [number];
+    yCoordinatesMap: [number];
+    showTimestampControls: boolean;
+    imageUploaded: boolean;
+    imagePath: string;
+    assets: Asset[];
+}
+
+
+export class OrImageWidget implements OrWidgetEntity {
+
+    readonly DISPLAY_MDI_ICON: string = "file-image-marker"; // https://materialdesignicons.com;
+    readonly DISPLAY_NAME: string = "Image";
+    readonly MIN_COLUMN_WIDTH: number = 2;
+    readonly MIN_PIXEL_HEIGHT: number = 0;
+    readonly MIN_PIXEL_WIDTH: number = 0;
+
+    getDefaultConfig(widget: DashboardWidget): OrWidgetConfig {
+        return {
+            displayName: widget?.displayName,
+            attributeRefs: [],
+            xCoordinatesMap: [],
+            yCoordinatesMap: [],
+            showTimestampControls: false,
+            imageUploaded: false,
+            imagePath: "https://home3ds.com/wp-content/uploads/2018/11/PNG.png",
```

Comment:
I'm guessing we want to remove this when merging.
However I could imagine a default image would be nice to have.
If so, putting the PNG file in one of the `resources` folders (don't know which one) is preferred.